slides/git-intro/index.html

```diff
@@ -455,17 +522,11 @@
         <script>
             var reveal = 0;
             var totalElements = document.querySelectorAll('.code-comment').length;
-            var offset = Math.floor(document.getElementById("recap").offsetHeight / totalElements) - 2;
             document.addEventListener('keydown', function(event) {
-
-                console.log(totalElements)
-
                 if(event.keyCode == 85) {
                     if(reveal < totalElements) {
                         document.getElementById(reveal.toString()).style.display = 'block';
-                        document.getElementById(reveal.toString()).style.top = offset + 'px';
                         reveal++;
-                        offset += Math.floor((document.getElementById("recap").offsetHeight / totalElements) - 0.5);
                     }
                 }
             });
```

Comment:
@michael-martin This **should** fix the alignment issue in the recap session...... **SHOULD.**

If it doesn't, let me know. I'll prolly think of a cleaner way to do it at the weekend to make it easier to use the feature in the future, but this is still a reasonable valid solution 😂